app/src/main/java/com/example/pantrypal/Comment.java

```diff
@@ -1,33 +1,51 @@
 package com.example.pantrypal;
 
 public class Comment {
+    private String id;
     private String title;
     private String message;
 
-    Comment(){
+    Comment() {
```

Comment:
Could you make the methods public? The default in Java is package-private which is weird